packages/extension-chakra-storefront/src/components/dynamic-image/index.jsx

```diff
@@ -6,7 +6,8 @@
  */
 import React, {useMemo} from 'react'
 import PropTypes from 'prop-types'
-import {Img, Box, useTheme} from '@chakra-ui/react'
+import {Image, Box} from '@chakra-ui/react'
+import {breakpoints} from '../../theme'
```

Comment:
FYI: breakpoints objects is no longer accessible in V3. So we have explicitly defined it in the theme